pages/events.jsx

```diff
@@ -108,70 +192,99 @@ var EventsPage = React.createClass({
+          <section className="join-global-movement">
             <Illustration
             height={183} width={156}
             src1x="/img/pages/events/svg/maker-party-logo.svg"
             alt="Maker Party logo"
+            className="content-first"
             >
-              <h2>
-                Join us for the worldwide Maker Party on July 15-31, 2015.
-              </h2>
+              <h2>Join the Global Movement</h2>
+              <p>Since its inauguration in 2012, Maker Party has become Mozilla's largest celebration of making and learning on the web. From getting the hang of HTML to building robots to learning about remixing using paper and scissors, people of all ages and from all backgrounds have come together to joyfully explore the culture, mechanics and citizenship of the web.</p>
             </Illustration>
+          </section>
+          <section>
             <div className="row">
               <div className="col-sm-12 col-md-12 col-lg-12">
+                <h2>What is a Maker Party?</h2>
                 <div className="video-container">
-                  <iframe src="https://www.youtube.com/embed/oko6TzPQE6Y" frameBorder="0" allowFullScreen className="video"></iframe>
+                  <iframe src="https://www.youtube.com/embed/oko6TzPQE6Y" frameBorder="0" allowFullScreen className="video" title="Maker Party Video"></iframe>
                 </div>
               </div>
             </div>
-            <div className="row">
-              <div className="col-xs-4 col-sm-4 col-md-4 col-lg-4">
-                <a href="https://www.flickr.com/photos/mozilladrumbeat/14592256638/in/set-72157647305286970"><img alt="2014 Hive NYC Summer Quest Maker Party" src="//c3.staticflickr.com/3/2915/14592256638_55ec5cf36b_b.jpg" className="flickr-thumbnail" /></a>
-              </div>
-              <div className="col-xs-4 col-sm-4 col-md-4 col-lg-4">
-                <a href="https://www.flickr.com/photos/mozilladrumbeat/15213220651/in/album-72157647305286970/"><img alt="Appmaking" src="//c2.staticflickr.com/6/5587/15213220651_9055dffbe6.jpg" className="flickr-thumbnail" /></a>
-              </div>
-              <div className="col-xs-4 col-sm-4 col-md-4 col-lg-4">
-                <a href="https://www.flickr.com/photos/newyouthcity/14715964794/in/faves-mozilladrumbeat/"><img alt="2014 Hive NYC Summer Quest Maker Party" src="//c4.staticflickr.com/4/3905/14715964794_a9fa75002c_c.jpg" className="flickr-thumbnail" /></a>
-              </div>
+          </section>
+        </div>
+        <div className="row mp-activities-banner">
+          <section>
+            <div className="btn-container">
+              <a className="btn btn-awsm" href="">Get the 2015 Maker Party Activities</a>
```

Comment:
waiting for https://github.com/mozilla/teach.webmaker.org/pull/1048 to land
and I'll change this to `<Link to="maker-party-2015">`